StaticAnalysisQualityGate/task.json

```diff
@@ -0,0 +1,42 @@
+{
+  "id": "e6ed21fe-1676-4db1-8826-9bae3e37ec5c",
+  "name": "ParasoftStaticAnalysisQualityGate",
+  "friendlyName": "Parasoft Static Analysis Quality Gate",
+  "description": "Check Parasoft Static Analysis Quality Gate",
+  "helpMarkDown": "[More Information](https://www1.parasoft.com/doc/link?product=findingsVSTS010)",
+  "category": "Test",
+  "visibility": [
+    "Build"
+  ],
+  "author": "Parasoft Corporation",
+  "version": {
+    "Major": 1,
+    "Minor": 1,
+    "Patch": 4
+  },
+  "minimumAgentVersion": "2.144.0",
+  "groups": [
+    {
```

Comment:
No groups any more beacuse we consider to seperate code coverage and static analysis.
It is better to remove the group and input, since we only need to use the default content.

Reply:
This is a placeholder UI, this group will be removed when implement configuration UI.